static-get

```diff
@@ -369,7 +370,26 @@ _static_get() {
                 if [ "${_sget__hash}" = "${_sget__hash_new}" ]; then
                     _cat "${tmpcache}.${uniq_id_mirror}/${_sget__bfile}"
 
-                    if [ "${extract}" ]; then
+                    if [ "${install}" ]; then
+                        _mkdir_p "${directory}/${_sget__bfile%%.tar*}"
+                        (cd "${directory}/${_sget__bfile%%.tar*}" && \
+                        ${compress_bin} < "../${_sget__bfile}"  | tar xf -) && \
```

Comment:
Not really sure about the `&& \` at the end of this line, I think it's unclear at best